test/browsers/browsers-target.js

```diff
@@ -0,0 +1,15 @@
+// See https://wiki.saucelabs.com/display/DOCS/Platform+Configurator# for a list of supported options
+const browsers = [
+    { 'platform': 'Windows 10', 'browserName': 'chrome', 'version': 'latest' },
+    { 'platform': 'Windows 10', 'browserName': 'firefox', 'version': 'latest' },
+    { 'platform': 'Windows 10', 'browserName': 'MicrosoftEdge', 'version': 'latest' }
+
+    // { 'platform': 'macOS 10.14', 'browserName': 'chrome', 'version': 'latest' },
```

Comment:
Since these lines are commented because there's still some work to be done as you say in [this comment](https://github.com/CartoDB/carto-vl/pull/1245#issuecomment-466335161), I'd add a `// FIXME` or `// TODO` comment explaining it. What do you think?

Reply:
For macOS: https://github.com/CartoDB/carto-vl/issues/1272
For Linux, it looks like the SauceLabs support is on very old browsers, not useful for us, so we should remove them

Reply:
Done!